server/index.js

```diff
@@ -40,5 +41,4 @@ app.get('/*', function(req, res){
   res.sendFile( assetFolder + '/index.html' )
 })
 
-app.listen(port)
```

Comment:
The app was crashing when I tried to run it, but if this line in server/index.js is removed, it works. I think this might be because there's already an app.listen() in Sockets.js